### PR TITLE
Replace sign up for free with book a demo

### PR DESCRIPTION
**Pull request by SteveRosam**

### CODE REVIEW

File: docs/connectors/index.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ Connectors are part of our [open source](https://github.com/quixio/quix-samples)
 
 Connectors help our users connect with other vendors such as AWS and Kafka.
 
-You can explore the connector README files here in Quix Docs. When you are ready to start using them, head over to the Quix Code Samples [GitHub](https://github.com/quixio/quix-samples){target="_blank"} repository, or [sign up](https://quix.io/signup){target="_blank"} and [log in to the platform](https://portal.platform.quix.ai/){target="_blank"}.
+You can explore the connector README files here in Quix Docs. When you are ready to start using them, head over to the Quix Code Samples [GitHub](https://github.com/quixio/quix-samples){target="_blank"} repository, or A Quix Cloud account. [Book a demo with us](https://meetings.hubspot.com/mike-rosam/product-demo){target="_blank"}.
```
Comment (peter-quix) [inline docs/connectors/index.md]: casing

**Review by peter-quix — DISMISSED** [review]

**Comment by merlin-quix** [comment]
Make it clearer that a demo is a prerequisite for a cloud account. Generalize "demo" to "session" I.e. If I'm reading the docs I probably know a bit about the prodict already-

````markdown
To use the APIs you'll typically need the following:

1. A Quix Cloud account. 
   To get a cloud account, [book a session ](https://meetings.hubspot.com/mike-rosam/product-demo) with a Quix representative.

4. Obtain a Personal Access Token - you need this to authenticate requests.
````

File: docs/kb/glossary.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ The [Query API](../apis/query-api/overview.md) is used to query persisted data.
 
 Quix provides an online Integrated Development Environment (IDE) for Python projects. When you open any project, you will see the **Run** button, and a console during runtime, in addition to the IntelliSense. It also provides general UI for deploying your code, and managing your projects and environments.
 
-Sign up for a [free account](https://portal.platform.quix.io/self-sign-up).
+[Book a demo with us](https://meetings.hubspot.com/mike-rosam/product-demo) to get a free account.
```
Comment (stereosky) [inline docs/kb/glossary.md]:
```suggestion
[Book a demo with us](https://meetings.hubspot.com/mike-rosam/product-demo) to start for free.
```

File: docs/apis/portal-api/setup.md [hunk 3]
```diff
@@ -2,7 +2,12 @@
 
 To use the APIs you'll typically need the following:
 
-1. [Sign up to Quix](https://portal.platform.quix.io/self-sign-up).
+1. A Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/apis/portal-api/setup.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/apis/streaming-reader-api/setup.md [hunk 4]
```diff
@@ -2,7 +2,12 @@
 
 To use the APIs you'll typically need the following:
 
-1. [Sign up to Quix](https://portal.platform.quix.io/self-sign-up).
+1. A Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/apis/streaming-reader-api/setup.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/apis/streaming-writer-api/setup.md [hunk 5]
```diff
@@ -2,7 +2,12 @@
 
 To use the APIs you'll typically need the following:
 
-1. [Sign up to Quix](https://portal.platform.quix.io/self-sign-up).
+1. A Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/apis/streaming-writer-api/setup.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/connectors/index.md [hunk 6]
```diff
@@ -9,6 +9,11 @@ Connectors are part of our [open source](https://github.com/quixio/quix-samples)
 
 Connectors help our users connect with other vendors such as AWS and Kafka.
 
-You can explore the connector README files here in Quix Docs. When you are ready to start using them, head over to the Quix Code Samples [GitHub](https://github.com/quixio/quix-samples){target="_blank"} repository, or [sign up](https://quix.io/signup){target="_blank"} and [log in to the platform](https://portal.platform.quix.ai/){target="_blank"}.
+You can explore the connector README files here in Quix Docs. When you are ready to start using them, head over to the Quix Code Samples [GitHub](https://github.com/quixio/quix-samples){target="_blank"} repository, or get a Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/connectors/index.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/create/create-project.md [hunk 7]
```diff
@@ -88,7 +88,12 @@ This page describes how to create a new project, and populate it with two enviro
 
 To do anything useful with Quix, you'll need at least one project, and one environment. You can think of a project as corresponding to a Git repository, and an environment as corresponding to Git branch within that repository. 
 
-1. [Sign up](https://portal.platform.quix.io/self-sign-up){target=_blank} and log into Quix.
+1. Login in to Quix Cloud.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/create/create-project.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/develop/integrate-data/jupyter-nb.md [hunk 8]
```diff
@@ -14,7 +14,12 @@ Although Quix is a real-time platform, to build real-time in-memory models and d
 
 You'll need some data stored in Quix. You can use any of the Quix [data sources](../../connectors/index.md) available in the Quix Code Samples. 
 
-You can also follow the onboarding process when you [sign-up to Quix](https://portal.platform.quix.io/self-sign-up?xlink=docs){target=_blank}. This process helps you create a source.
+You can also follow the onboarding process after you login for the first time. The onboarding process helps you create your first `source`.
+
+??? info "Start for free"
+	[Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+	
+	We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/develop/integrate-data/jupyter-nb.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/integrations/databases/influxdb/quickstart.md [hunk 9]
```diff
@@ -21,7 +21,13 @@ This quickstart demonstrates both saving data to InfluxDB and querying data from
 
 To complete this quickstart you will need:
 
-* A free [Quix account](https://portal.platform.quix.ai/self-sign-up){target="_blank"}.
+* A Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/integrations/databases/influxdb/quickstart.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/integrations/databases/influxdb/telegraf.md [hunk 10]
```diff
@@ -13,7 +13,13 @@ search:
 
 This guide assumes you have the following:
 
-* A free [Quix account](https://portal.platform.quix.ai/self-sign-up){target="_blank"}.
+* A Quix Cloud account.
+
+  ??? info "Start for free"
+      [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+      
+      We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/integrations/databases/influxdb/telegraf.md]:
```suggestion
      We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/kb/glossary.md [hunk 11]
```diff
@@ -171,7 +171,10 @@ The [Query API](../apis/query-api/overview.md) is used to query persisted data.
 
 Quix provides an online Integrated Development Environment (IDE) for Python projects. When you open any project, you will see the **Run** button, and a console during runtime, in addition to the IntelliSense. It also provides general UI for deploying your code, and managing your projects and environments.
 
-Sign up for a [free account](https://portal.platform.quix.io/self-sign-up).
+??? info "Start for free"
+	[Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+	
+	We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/kb/glossary.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/kb/what-is-quix.md [hunk 12]
```diff
@@ -15,9 +15,10 @@ Such intelligent real-time decision making has many use cases, including example
 
 Quix has excellent synergy with Machine Learning (ML) systems too. You can quickly deploy your ML model and monitor its performance in real time, modify the model, and redeploy it with a single click. 
 
-!!! tip
-
-    [Sign up for free](https://portal.platform.quix.io/self-sign-up){target=_blank}.
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/kb/what-is-quix.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/quix-cloud/quickstart.md [hunk 13]
```diff
@@ -19,16 +19,14 @@ If you're new to Quix Cloud or need a refresher on its features, [learn more her
...
     Before proceeding, ensure you've completed the [Quix CLI Quickstart](../quix-cli/cli-quickstart.md).
 
-## Step 1: Sign up to Quix Cloud
+## Step 1: Sign in to Quix Cloud
 
-It's free to [sign up to Quix Cloud](https://portal.platform.quix.io/self-sign-up){target=_blank}.
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+        
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/quix-cloud/quickstart.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/quix-cloud/quixtour/ingest.md [hunk 14]
```diff
@@ -2,9 +2,12 @@
...
 In this part of the tour you'll get some data flowing into your pipeline. You do this by reading CPU load on your service and publishing it to a topic.
 
-## Step 1: Sign up to Quix Cloud
+## Step 1: Sign in to Quix Cloud
 
-It's free to [sign up to Quix Cloud](https://portal.platform.quix.io/self-sign-up){target=_blank}.
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/quix-cloud/quixtour/ingest.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/clickstream/overview.md [hunk 15]
```diff
@@ -51,7 +51,12 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 ## Prerequisites
 
-To get started make sure you have a [free Quix account](https://portal.platform.quix.ai/self-sign-up).
+You'll need a Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/clickstream/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/computer-vision/overview.md [hunk 16]
```diff
@@ -62,7 +62,12 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 ## Prerequisites
 
-To get started make sure you have a [free Quix account](https://portal.platform.quix.io/self-sign-up).
+You'll need a Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/computer-vision/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/currency-alerting/currency-alerting.md [hunk 17]
```diff
@@ -22,7 +22,13 @@ If you need any help, please sign up to the [Quix Community](https://quix.io/sla
 
 To complete this tutorial you will need the following accounts:
 
-* [Quix](https://quix.io){target=_blank} - You can sign up for a free account [here](https://portal.platform.quix.io/self-sign-up){target=_blank}. This enables you to create the real-time stream processing pipeline.
+* [Quix Cloud account](https://quix.io){target=_blank} - This enables you to create the real-time stream processing pipeline.
+
+??? info "Start for free"
+      [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+      
+      We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/currency-alerting/currency-alerting.md]:
```suggestion
      We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/data-stream-processing/data-stream-processing.md [hunk 18]
```diff
@@ -46,10 +46,12 @@ You have to create and deploy three applications. Quix has:
 
 - A Quix account
 
-- A mobile device
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/data-stream-processing/data-stream-processing.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/influxdb-alerting/overview.md [hunk 19]
```diff
@@ -24,7 +24,13 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 To complete this tutorial you'll need to:
 
-* [Obtain a free Quix account](https://portal.platform.quix.io/self-sign-up)
+* Get a Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/influxdb-alerting/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/influxdb-migration/overview.md [hunk 20]
```diff
@@ -25,7 +25,13 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 To complete this tutorial you'll need to:
 
-* [Obtain a free Quix account](https://portal.platform.quix.io/self-sign-up)
+* Get a Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/influxdb-migration/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/matlab/matlab-and-simulink.md [hunk 21]
```diff
@@ -14,7 +14,13 @@ This section describes the steps for deploying a MATLAB function that rotates 2D
 
 ### Prerequisites
 
-* A Quix account. You can sign up for a free account from the Quix [website](https://quix.io/product/){target=_blank}.
+* A Quix account. 
+
+??? info "Start for free"
+	[Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+	
+	We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/matlab/matlab-and-simulink.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/matlab/matlab-and-simulink.md [hunk 22]
```diff
@@ -127,7 +133,13 @@ This section describes the steps for deploying a Simulink model of an [internal
 
 ### Prerequisites
 
- - A Quix account. You can sign up for a free account from the Quix [website](https://quix.io/product/){target=_blank}.
+ - A Quix Cloud account. 
+ 
+??? info "Start for free"
+	[Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+	
+	We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/matlab/matlab-and-simulink.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/predictive-maintenance/overview.md [hunk 23]
```diff
@@ -43,7 +43,12 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 ## Prerequisites
 
-To get started make sure you have a [free Quix account](https://portal.platform.quix.ai/self-sign-up){target="_blank"}.
+You'll need a Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/predictive-maintenance/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/rss-tutorial/rss-processing-pipeline.md [hunk 24]
```diff
@@ -22,8 +22,12 @@ This tutorial has three parts
...
 What you need
 
-  - A free [Quix account](https://quix.io/signup){target=_blank}. It
-    comes with enough credits to create this application.
+  - A Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/rss-tutorial/rss-processing-pipeline.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/sentiment-analysis/overview.md [hunk 25]
```diff
@@ -60,7 +60,12 @@ If you need any assistance while following the tutorial, we're here to help in t
 
 ## Prerequisites
 
-To get started make sure you have a [free Quix account](https://portal.platform.quix.io/self-sign-up).
+Get a Quix Cloud account.
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/sentiment-analysis/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/slack-alerting/slack-alerting.md [hunk 26]
```diff
@@ -38,7 +38,12 @@ A Quix Service receives data from the Transport For London BikePoint API and str
 
 To proceed with this tutorial you need:
 
- - [Free Quix account](https://portal.platform.quix.io/self-sign-up/){target=_blank}.
+ - A Quix Cloud account. 
+
+??? info "Start for free"
+	[Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+	
+	We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/slack-alerting/slack-alerting.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/tutorials/train-and-deploy-ml/overview.md [hunk 27]
```diff
@@ -16,7 +16,13 @@ If you'd like to watch a video before stepping through this tutorial, you can vi
 
 To complete this tutorial you need the following:
 
-* A free [Quix account](https://portal.platform.quix.io/self-sign-up/){target=_blank}.
+* A Quix Cloud account. 
+
+??? info "Start for free"
+    [Book a session](https://meetings.hubspot.com/mike-rosam/product-demo) with us to start for free.
+    
+    We will create a time limited free account for you, and our experts will help you get started with your specific use case.
```
Comment (stereosky) [inline docs/tutorials/train-and-deploy-ml/overview.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/develop/integrate-data/jupyter-nb.md (hunk 8 above)
Comment (stereosky) [inline docs/develop/integrate-data/jupyter-nb.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/integrations/databases/influxdb/quickstart.md (hunk 9 above)
Comment (stereosky) [inline docs/integrations/databases/influxdb/quickstart.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/integrations/databases/influxdb/telegraf.md (hunk 10 above)
Comment (stereosky) [inline docs/integrations/databases/influxdb/telegraf.md]:
```suggestion
      We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/kb/glossary.md (hunk 11 above)
Comment (stereosky) [inline docs/kb/glossary.md]:
```suggestion
	We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/kb/what-is-quix.md (hunk 12 above)
Comment (stereosky) [inline docs/kb/what-is-quix.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

File: docs/quix-cloud/quickstart.md (hunk 13 above)
Comment (stereosky) [inline docs/quix-cloud/quickstart.md]:
```suggestion
    We will create a time-limited free account for you, and our experts will help you get started with your specific use case. 
```

**Review by stereosky — APPROVED** [review]

**Pull request merged**